doc-deploy-dev-index/action.yml

```
@@ -0,0 +1,139 @@
+    default: 'documentation-html'
+    type: string
+
+  template:
+    description: >
+      The "template" parameter specifies the layout used for the HTML documentation.
+      By default, it is set to ``sphinx_pydata`` which assumes that the document is
+      constructed using the pydata-sphinx-theme or its associated theme,
+      such as the ansys-sphinx-theme.
+    required: false
+    default: "sphinx_pydata"
+    type: string
+
+
+
+runs:
+  using: "composite"
+  steps:
+
+    # ------------------------------------------------------------------------
+
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Checkout the repository branch for deploying the documentation. If
+          this step fails, then it means that the provided token is not valid.
+
+    - name: "Get the name of the repository"
+      shell: bash
+      run: |
+        if [[ "${{ inputs.repository }}" == "current" ]]; then
+          echo "REPOSITORY=${{ github.repository }}" >> $GITHUB_ENV
+        else
+          echo "REPOSITORY=${{ inputs.repository }}" >> $GITHUB_ENV
+        fi
+
+    - name: "Checkout ${{ env.REPOSITORY }} repository"
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ env.REPOSITORY }}
+        ref: ${{ inputs.branch }}
+        token: ${{ inputs.token }}
+
+    # ------------------------------------------------------------------------
+
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Download the documentation artifact from the current workflow.
+
+    - name: "Download the development documentation artifact"
+      uses: actions/download-artifact@v3
+      with:
+        name: ${{ inputs.doc-artifact-name }}
+        path: ${{ inputs.doc-artifact-name }}
+
+    - name: "Display structure of ${{ inputs.doc-artifact-name }}"
+      shell: bash
+      run: |
+        ls -R ${{ inputs.doc-artifact-name }}
+
+    # ------------------------------------------------------------------------
+
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Install the pymeilisearch required for the indexing.
+
+    - name: "Set up Python"
+      uses: ansys/actions/_setup-python@main
+      with:
+        python-version: '3.10'
+
+    - name: "Install build and twine"
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip pymeilisearch
+
+     # ------------------------------------------------------------------------
+
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Scrap the document and deploy it to pymeilisearch.
+
+    - name: Scrape the dev documentation to meilisearch
+      shell: bash
+      run: |
+        pymeilisearch upload --template ${{ inputs.template }} --index ${{ inputs.library_name }}-vdev  --cname ${{ inputs.cname }}/version/dev/ html ${{ inputs.doc-artifact-name }}
```

No need to add `v` in front of the `dev` suffix:

```suggestion
        pymeilisearch upload --template ${{ inputs.template }} --index ${{ inputs.library-name }}-dev  --cname ${{ inputs.cname }}/version/dev/ html ${{ inputs.doc-artifact-name }}
```